tests/components/setOrderEntry/OrderSetForm.test.jsx

```diff
@@ -26,6 +26,13 @@ describe('Component: OrderSetForm', () => {
     const renderedComponent = getComponent().instance();
     const { changeSelectedOrderSet } = renderedComponent;
     changeSelectedOrderSet();
-    expect(renderedComponent.state.displaySets).toEqual('none');
+    expect(renderedComponent.state.displaySets).toEqual(false);
+  });
+
+  it('should hide the segment of the form that displays the order details when cancelOrderSet is called', () => {
+    const renderedComponent = getComponent().instance();
+    const { cancelOrderSet } = renderedComponent;
+    cancelOrderSet();
+    expect(renderedComponent.state.displaySets).toEqual(true);
```

Comment:
I think as previously stated from your test description, it should hide the form. This should not test that the `displaySet` is present but the form is hidden so a suggestion is:
```suggestion
    expect(renderedComponent.state.displayForm).toEqual(false);
```